src/gui/gtkentry.c

```diff
@@ -169,69 +169,78 @@ void dt_gtkentry_setup_completion(GtkEntry *entry, const dt_gtkentry_completion_
 {
   static dt_gtkentry_completion_spec default_path_compl_list[]
-      = { { "ROLL_NAME", N_("$(ROLL_NAME) - roll of the input image") },
-          { "FILE_FOLDER", N_("$(FILE_FOLDER) - folder containing the input image") },
-          { "FILE_NAME", N_("$(FILE_NAME) - basename of the input image") },
-          { "FILE_EXTENSION", N_("$(FILE_EXTENSION) - extension of the input image") },
+      = { { "ROLL.NAME", N_("$(ROLL.NAME) - roll of the input image") },
+          { "FILE.FOLDER", N_("$(FILE.FOLDER) - folder containing the input image") },
+          { "FILE.NAME", N_("$(FILE.NAME) - basename of the input image") },
+          { "FILE.EXTENSION", N_("$(FILE.EXTENSION) - extension of the input image") },
           { "VERSION", N_("$(VERSION) - duplicate version") },
-          { "VERSION_IF_MULTI", N_("$(VERSION_IF_MULTI) - same as $(VERSION) but null string if only one version exists") },
-          { "VERSION_NAME", N_("$(VERSION_NAME) - version name from metadata") },
+          { "VERSION.IF_MULTI", N_("$(VERSION.IF_MULTI) - same as $(VERSION) but null string if only one version exists") },
+          { "VERSION.NAME", N_("$(VERSION.NAME) - version name from metadata") },
           { "JOBCODE", N_("$(JOBCODE) - job code for import") },
           { "SEQUENCE", N_("$(SEQUENCE) - sequence number") },
-          { "MAX_WIDTH", N_("$(MAX_WIDTH) - maximum image export width") },
-          { "MAX_HEIGHT", N_("$(MAX_HEIGHT) - maximum image export height") },
-          { "SENSOR_WIDTH", N_("$(SENSOR_WIDTH) - image sensor width") },
-          { "SENSOR_HEIGHT", N_("$(SENSOR_HEIGHT) - image sensor height") },
-          { "RAW_WIDTH", N_("$(RAW_WIDTH) - RAW image width") },
-          { "RAW_HEIGHT", N_("$(RAW_HEIGHT) - RAW image height") },
-          { "CROP_WIDTH", N_("$(CROP_WIDTH) - image width after crop") },
-          { "CROP_HEIGHT", N_("$(CROP_HEIGHT) - image height after crop") },
-          { "EXPORT_WIDTH", N_("$(EXPORT_WIDTH) - exported image width") },
-          { "EXPORT_HEIGHT", N_("$(EXPORT_HEIGHT) - exported image height") },
+          { "WIDTH.MAX", N_("$(WIDTH.MAX) - maximum image export width") },
+          { "WIDTH.SENSOR", N_("$(WIDTH.SENSOR) - image sensor width") },
+          { "WIDTH.RAW", N_("$(WIDTH.RAW) - RAW image width") },
+          { "WIDTH.CROP", N_("$(WIDTH.CROP) - image width after crop") },
+          { "WIDTH.EXPORT", N_("$(WIDTH.EXPORT) - exported image width") },
+          { "HEIGHT.MAX", N_("$(HEIGHT.MAX) - maximum image export height") },
+          { "HEIGHT.SENSOR", N_("$(HEIGHT.SENSOR) - image sensor height") },
+          { "HEIGHT.RAW", N_("$(HEIGHT.RAW) - RAW image height") },
+          { "HEIGHT.CROP", N_("$(HEIGHT.CROP) - image height after crop") },
+          { "HEIGHT.EXPORT", N_("$(HEIGHT.EXPORT) - exported image height") },
           { "YEAR", N_("$(YEAR) - year") },
+          { "YEAR.SHORT", N_("$(YEAR.SHORT) - year without century") },
```

Comment:
And YEAR.LONG here too?

Reply:
Idem, but I'll would be yet more cautious here, as that's the visible dictionary. So far we don't have synonym. If we start, there is more difficult not to put also all the others.

Reply:
Ok, let's this as-is at least for now. Thanks @phweyland for all the work you put on this.